### PR TITLE
added BankingApp class

### PR DESCRIPTION
**Pull request by rukshand2001**

### CODE REVIEW

File: src/main/java/org/example/BankingApp.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    private static ArrayList<Account> accounts = new ArrayList<Account>();
+    private static final double INTEREST_RATE = 0.1;
+
+
+    public static void main(String[] args) {
+
+        BankingApp bankingApp = new BankingApp();
+        bankingApp.mainUI();
+    }
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java:17]: Add a new line here

File: src/main/java/org/example/BankingApp.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        System.out.println("*********************************\n\n");
+
+        while (true){
+
+            System.out.println("Press 1 to Logging to App");
+            System.out.println("Press 2 to Create Account");
+            System.out.println("Press 3 to  Exit");
+
+            System.out.print("\nPlease enter your choice : ");
+            int enterChoice = scanner.nextInt();
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]:
```suggestion
            int choice = scanner.nextInt();
```

File: src/main/java/org/example/BankingApp.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+            if(enterChoice == 1){
+
+
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]: remove extra lines and call loggingUI function here

File: src/main/java/org/example/BankingApp.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        newAccount.setAccountBalance(initialBalance);
+        newAccount.setInterestRate(BankingApp.INTEREST_RATE);
+
+        BankingApp.accounts.add(newAccount);
+    }
+
+    public void loggingUI(Scanner scanner){
+        /*Logging to Account*/
+
+        Account newLoggingAccount = new Account();
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]: we don't need to create an account

File: src/main/java/org/example/BankingApp.java [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+        while (true){
+            System.out.print("\n Enter your account number : ");
+            int accountNumber = scanner.nextInt();
+            System.out.print("Enter Your Name : ");
+            String accountHoldersName = scanner.next();
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]: Why do you need account holders name in login ?

File: src/main/java/org/example/BankingApp.java [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+            for (int i=0;i<BankingApp.accounts.size();i++){
+                if(accountNumber == BankingApp.accounts.size()+1){
+                    System.out.println("Well Come "+accountHoldersName+" Your Account Number is "+accountNumber);
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]: wrong logic !

File: src/main/java/org/example/BankingApp.java (hunk 1 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java:17]: done

File: src/main/java/org/example/BankingApp.java (hunk 2 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: done

File: src/main/java/org/example/BankingApp.java (hunk 3 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: done

File: src/main/java/org/example/BankingApp.java (hunk 4 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: done

File: src/main/java/org/example/BankingApp.java (hunk 5 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: remove

File: src/main/java/org/example/BankingApp.java (hunk 6 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: changed

**Review by aselarbd — APPROVED** [review]

**Pull request merged**